app/server/utils/ussd/token_processor.py

```diff
@@ -23,28 +23,31 @@ def send_sms(user, message_key, **kwargs):
     @staticmethod
     def send_success_sms(message_key: str, amount: float, user: User, other_user: User, reason: str, tx_time: datetime,
                          balance: float):
-        # TODO: convert amounts - truncate? need to turn from cents to dollar?
-        TokenProcessor.send_sms(user, message_key, amount=amount, token_name=default_token(user),
+        rounded_balance = str(round(balance, 2))
```

Comment:
decided to round everything, otherwise it looks terrible with a bunch of decimal places